Resources/Locale/en-US/prototypes/catalog/fills/crates/livestock-crates.ftl

```diff
@@ -53,4 +53,7 @@ ent-CrateNPCLizard = Lizard crate
     .desc = A crate containing a lizard.
 
 ent-CrateNPCKangaroo = Kangaroo crate
-    .desc = A crate containing a kangaroo.
\ No newline at end of file
+    .desc = A crate containing a kangaroo.
+
+ent-CrateNPCMothroach = Crate of mothroaches
+    .desc = A crate containing four mothroaches. You can hear faint chittering inside.
```

Comment:
From what I'm understanding, it seems like the extra sentence is throwing it off, somehow. Just change line 59 to be `.desc = A crate containing four mothroaches.` and make line 60 empty, and I think that might fix it.